src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
```

Comment:
`ballNumber` should be private because there is no need for any other class to access this field directly. All interaction with this field should be done via getter and setter.